### DISCUSSION TITLE
blog/note-taking-is-new-electronic-games

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#32] [Announcements]
# blog/note-taking-is-new-electronic-games

哲学片段与学术之路

https://pathos.page/blog/note-taking-is-new-electronic-games

**Comment by lg2it** [comment]
读罢想到去年写过一篇文章[《折腾笔记软件是把双刃剑》](https://justgoidea.com/posts/2023-013/)。我自己也有这样的时候，折腾来折腾去忘记了折腾的目的是什么。
